apps/api/.env.example

```
@@ -1,21 +1,15 @@
-API_KEY=****
-SUPER_ADMIN_API_KEY=****
-NEXT_PUBLIC_GA_MEASUREMENT_ID=****
-TEST_DATABASE_URL=****
-DATABASE_URL=****
-# Comma-separated list of webhook URLs to notify on map data changes (optional)
-# Format: https://webhook1.com/endpoint,https://webhook2.com/endpoint
-NOTIFY_WEBHOOK_URLS_COMMA_SEPARATED=
\ No newline at end of file
+NEXT_PUBLIC_AUTH_URL=****
+NEXT_PUBLIC_CHANNEL=****
+NEXT_PUBLIC_MAP_URL=****
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)